Dockerfile

```diff
@@ -8,7 +8,10 @@ ARG GIT_PYTHON_REFRESH=quiet
 # install extra requirements
 RUN apt-get clean
 RUN apt-get update -y
-RUN apt-get install gcc g++ libgeos-dev -y git -y pkg-config -y libhdf5-dev
+RUN apt-get install gcc g++ cmake libgeos-dev -y git -y pkg-config -y libhdf5-dev
+
+RUN git clone --recursive https://github.com/dmlc/xgboost
+RUN cd xgboost && git checkout 603f8ce2fa71eecedadd837316dcac95ab7f4ff7 && mkdir build && cd build && cmake .. && make -j4 && cd .. && cd python-package && pip install .
 
```

Comment:
Could you add a comment here, about once Xgboast 2.0 is out, we can change this back